client/coral-admin/src/AppRouter.js

```diff
@@ -47,18 +51,19 @@ const routes = (
         <Route path='rejected' components={Moderation}>
           <Route path=':id' components={Moderation} />
         </Route>
-        <Route path='flagged' components={Moderation}>
+        <Route path='reported' components={Moderation}>
           <Route path=':id' components={Moderation} />
         </Route>
-        <Redirect from=':id' to='all/:id' />
+
+        <Route path=':id' components={Moderation} />
+
         <IndexRedirect to='all' />
```

Comment:
This line should be removed as we don't want to automatically redirect to the "all" tab.